lib/msal-node/src/network/HttpClient.ts

```diff
@@ -10,6 +10,8 @@ import {
 } from "@azure/msal-common";
 import { HttpMethod } from "../utils/Constants";
 import axios, { AxiosRequestConfig } from "axios";
+import { HttpsProxyAgent } from "https-proxy-agent";
+import fetch from "node-fetch";
```

Comment:
So Axios doesn't support working with proxies? I find that surpising.

Reply:
Yeah. Its took a few escalations before we figured axios is the issue. :)